bin/tessel-2.js

```diff
@@ -197,6 +197,36 @@ parser.command('rename')
   .help('Change the name of a Tessel to something new.');
 
 
+// accessing the root shell of your tessels
+// Fixes issue https://github.com/tessel/t2-cli/issues/80
+/**
+$ t2 root --help
+> Usage: tessel root [-i <path>] [--help]
+>  -i <path>:   provide a path to the desired ssh key
+$ 
+
+$ t2 root
+> Accessing root...
+root@192.168.128.124 # 
+*/
+var default_id_rsa = '~/.tessel/id_rsa';
+var functional_msg = '\nGain SSH root access to one of your authorized tessels (menu listing if multiple targets)';
+parser.command('root')
+  .usage(functional_msg + '\n\nUsage: t2 root [-i PATH] [--help]\n\n-i PATH:   Optional targeting a different Private Key \n\n(Note: default target created by "t2 key generate" is ' + default_id_rsa + ')\n')
```

Comment:
The problem is, it would write `Usage: node t2 root` instead of `t2 root` - I like the idea of templates. 
@Frijol it looks like you are figuring out some things at https://github.com/tessel/t2-cli/pull/274 - is there also an general way to fix the `node t2 root` problem ?